source/codegen/metadata/visa/functions.py

```diff
@@ -1398,6 +1399,7 @@
             },
             {
                 'direction': 'out',
+                'include_in_proto': False,
```

Comment:
Optional: Seeing it in this context with the custom implementation reminded me of the tag `'return_on_error_key'`. What you have in the custom code looks good to me but alternatively you might be able to remove the `CustomCode` for this function and then have both `include_in_proto': False,` like you have here and also `'return_on_error_key': 'ni-failing-index'`.

It'd be interesting to see if that generates what you need. Though looking at the DAQmx usage of the key it seems to add the trailing metadata if there's an error or not instead of only when there's an error like in your custom implementation so that might be problematic: https://github.com/ni/grpc-device/blob/1e69024ec764188e78ff35d430618f918ffb0723/generated/nidaqmx/nidaqmx_service.cpp#L13070-13073

Reply:
That works well. Thanks!